apps/src/applab/GoogleChart.js

```diff
@@ -148,8 +148,9 @@ export default class GoogleChart {
    */
   static dataTableFromRowsAndColumns(rows, columns) {
     const dataArray = rows.map(row => columns.map(key => row[key]));
+    const columnLabels = columns.map(column => ({label: column}));
```

Comment:
I was running into errors with `invalid header type` for numeric columns, but looking at the docs, it seems like we were passing the column names in the wrong format anyways? https://developers.google.com/chart/interactive/docs/reference#arraytodatatable

Reply:
Possibly. We are also using an older version of the charts API, and I don't see a version on that documentation.  I'm fine with this change, but recommend you test the chart types in App Lab before and after this change to see if it affects anything surprising.

Reply:
Existing chart APIs appear unaffected:

![image](https://user-images.githubusercontent.com/8787187/68048907-cca66500-fc9e-11e9-8efb-d7ae1f5fe032.png)